Source/src/resources/ResourceTexture.h

```diff
@@ -58,7 +58,7 @@ namespace Hachiko
         unsigned mag_filter = 0x2601;
         unsigned wrap = 0x2900;
 
-        byte* data{};
+        ::byte* data{};
```

Comment:
What's the meaning of `::byte*` here?

Reply:
I had the same question and this is the solution of why is writed in that way
https://developercommunity.visualstudio.com/t/error-c2872-byte-ambiguous-symbol/93889?viewtype=solutions

Reply:
Noicee